clients/telegrafs/telegrafs.go

```diff
@@ -0,0 +1,187 @@
+package telegrafs
+
+import (
+	"context"
+	"errors"
+	"io/ioutil"
+	"os"
+
+	"github.com/influxdata/influx-cli/v2/api"
+	"github.com/influxdata/influx-cli/v2/clients"
+	"github.com/influxdata/influx-cli/v2/pkg/influxid"
+)
+
+type Client struct {
+	clients.CLI
+	api.TelegrafsApi
+}
+
+type telegrafPrintOpts struct {
+	graf  *api.Telegraf
+	grafs *api.Telegrafs
+}
+
+type ListParams struct {
+	clients.OrgParams
+	Id string
+}
+
+func (c Client) List(ctx context.Context, params *ListParams) error {
+	if !params.OrgID.Valid() || params.Id == "" {
+		return errors.New("at least one of org, org-id, or id must be provided")
+	}
+
+	if params.Id != "" {
+		id, err := influxid.IDFromString(params.Id)
```

Comment:
I don't think we need to do this conversion, since we're just calling `String()` on it below.